public/assets/css/style.css

```diff
@@ -193,6 +193,14 @@ h5 {
     background-color: rgba(0, 0, 0, 0.2);
 }
 
+.pro {
+    border: 2px solid black;
+    background-color: #3b3c3b;
+}
+
+.admin {
+    border: 2px solid black;
+    background-color: #9a7d6c;
```

Comment:
This CSS rule is not closed ... and that's the 2nd file on 40 ! Very good proofreading guys...